### PR TITLE
Adding wait_still_screen to make sure test goes right way

### PR DESCRIPTION
**Pull request by ostruzina**
A `wait_still_screen` step has been added into the `libreoffice_double_click_file` module.

Context:
The test opens nautilus and wants to loop through a list of document files of various types to check that they open without problems.
After checking the first one on the list (.doc), it wants to return back to nautilus (alt-tab) and proceed to the next document file (.docx).
However, the alt key seems to get released in a wrong moment; the test does not return to nautilus and instead stays in the .doc file (looks like tab has been pressed a couple more times), so the test cannot continue and fails.

Failed test: https://openqa.suse.de/tests/815852#step/libreoffice_double_click_file/87

The amended test went through fine on my local openQA instance (however it will need to be re-assessed in production to see how much of a help it is in an environment with higher workload):
http://dreamyhamster.suse.cz/tests/141#step/libreoffice_double_click_file/33

Related to poo#17758

### CODE REVIEW

File: tests/x11regressions/libreoffice/libreoffice_double_click_file.pm [hunk 1]
```diff
@@ -39,6 +39,7 @@ sub run() {
         if ($tag ne 'xlsx') {
             hold_key "alt";
             send_key_until_needlematch("libreoffice-nautilus-window", "tab");
+            wait_still_screen;
```
Comment (mitiao) [inline tests/x11regressions/libreoffice/libreoffice_double_click_file.pm]: LGTM.
And the default stilltime of wait_still_screen is 7s which is a little bit longer?
shorter time may work as well: wait_still_screen 3;

**Comment by mitiao** [comment]
There is a duplicated use base on top of the .pm:

use base "x11regressiontest";
use base "x11regressiontest";

It would be nice if you can remove the duplicated. Thanks.

**Comment by coolo** [comment]
your aventures in git are about to begin :)

**Comment by ostruzina** [comment]
Apologies for this mess, I guess I got too lost in rebasing branches.

I am going to close off this PR. I incorporated the suggestions brought forward by @mitiao on a new branch and made a new pull request #2575:

https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2575

@coolo: :see_no_evil:

**Pull request closed without merging**